closed/test/jdk/openj9/internal/security/TestPolicySunset.java

```diff
@@ -0,0 +1,180 @@
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        "", "",
+                        "",
+                        0),
+                // 7 - expired; suppress unset, ignore=true
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        null, "=true",
+                        "WARNING: java will start with the requested restricted security profile but uncertified cryptography may be active",
+                        0),
+                // 8 - expired; suppress=false; ignore=false
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        "=false", "=false",
+                        "Use -Dsemeru.restrictedsecurity.ignoresunsetexpiration to allow java to start while possibly using uncertified cryptography",
+                        1),
+                // 9 - expired; suppress=true; ignore=false, no warning
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        "=true", "=false",
+                        "",
+                        1),
+                // 10 - expired; suppress=true, ignore unset, no warning
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        "=true", null,
+                        "",
+                        1),
+                // 11 - expired; suppress=false; ignore unset
+                Arguments.of("Test-Profile-PolicySunset-Expired",
+                        propertyFile,
+                        "=false", null,
+                        "Use -Dsemeru.restrictedsecurity.ignoresunsetexpiration to allow java to start while possibly using uncertified cryptography",
+                        1));
+    }
+
+    @ParameterizedTest
+    @MethodSource("patternMatches_testPolicySunset")
+    public void shouldContain_testPolicySunset(String customprofile, String securityPropertyFile,
+            String suppresssunsetwarning, String ignoresunsetexpiration, String expected, int exitValue)
+            throws Exception {
+        List<String> args = new ArrayList<>();
+
+        args.add("-Dsemeru.fips=true");
+        args.add("-Dsemeru.customprofile=" + customprofile);
+        args.add("-Djava.security.properties=" + securityPropertyFile);
+        if (suppresssunsetwarning != null) {
+            args.add("-Dsemeru.restrictedsecurity.suppresssunsetwarning" + suppresssunsetwarning);
+        }
+        if (ignoresunsetexpiration != null) {
+            args.add("-Dsemeru.restrictedsecurity.ignoresunsetexpiration" + ignoresunsetexpiration);
+        }
+        args.add("TestPolicySunset");
+
+        OutputAnalyzer outputAnalyzer = ProcessTools.executeTestJava(args.toArray(new String[0]));
```

Comment:
Please just use the list directly (via `executeTestJava(List<String>)`:
```java
        OutputAnalyzer outputAnalyzer = ProcessTools.executeTestJava(args);
```

Reply:
Updated.